prow/monitoring/BUILD.bazel

```diff
@@ -56,3 +61,31 @@ k8s_objects(
       ":servicemonitor_crd",
    ]
 )
+
+load("//def:configmap.bzl", "k8s_configmap")
+
+k8s_configmap(
+   name = "grafana-datasources",
+   cluster = "{STABLE_PROW_CLUSTER}",
+   namespace = "prow-monitoring",
+   data = {
+       "datasources.yaml": ":datasources.yaml",
+   },
+)
+
+k8s_configmap(
+   name = "grafana-config",
+   cluster = "{STABLE_PROW_CLUSTER}",
+   namespace = "prow-monitoring",
+   data = {
+       "grafana.ini": ":grafana.ini",
+   },
+)
+
+k8s_objects(
```

Comment:
Please add these to the release target instead of defining a separate target. It is unintuitive that there is a release target that only deploys some of the components and separate targets have to be used to deploy the rest.

Reply:
I wanted to do that too but I do not know how to include non-component call into release target.
Could you please give me an example?

Reply:
Could we do `component("grafana", "datasources", "config")`?

Reply:
ConfigMaps can also be specified like this: https://github.com/kubernetes/test-infra/blob/5c432cee243f1162d0339fd879fe368d79e124b7/prow/cluster/starter.yaml#L13-L29

We could then use `component("grafana-datasources", "configmap")` and `component("grafana-config", "configmap")`.

Reply:
Oh, you mean to abandon the idea of using `k8s_configmap` ... but i think the point here is to keep the original file in the src so that we can edit it easily. Isnt it why k8s_config and k8s_secret rules exist? Maybe I did not understand it correctly.

It is just the issue of `component` and `k8s_xxx` in //def not well integrated.

Reply:
If we inline the config file in the configmap it's no harder to edit it, though

Reply:
OK ... inline is coming.

Reply:
Done.